syntax = "proto3";

// import "google/protobuf/timestamp.proto";

option objc_class_prefix = "RTG";

package Seq2SeqGRPC;

service Seq2SeqService {
	rpc RespondToText (ConversationInput) returns (ConversationResponse);
}

message ConversationInput {
	string trans_id = 1;
	string text = 2;
	// google.protobuf.Timestamp time_name = 3;
}

message ConversationResponse {
	string trans_id = 1;
	enum State {
		SUCCESS = 0;
		MODEL_ERR = 1;
		UNKNOWN = 2;
	}
	State state = 2;
	string text = 3;
	// google.protobuf.Timestamp time_name = 4;
}